packages/grafana-runtime/src/services/LocationService.tsx

```diff
@@ -153,10 +155,25 @@ export function locationSearchToObject(search: string | number): UrlQueryMap {
   return {};
 }
 
+const history = H.createBrowserHistory({ basename: config.appSubUrl ?? '/' });
 /**
  * @public
  */
-export let locationService: LocationService = new HistoryWrapper();
+export let locationService: LocationService = new HistoryWrapper(
+  createAggregateHistory({
+    actualHistory: history,
+    isMain: true,
+    param: '__sc',
+  })
+);
+
+export let locationServiceSecondary: LocationService = new HistoryWrapper(
```

Comment:
I don't like much this sort of hardcoding here but not sure how to do it otherwise while maintaining the functionality of the old locationService global var.